app/routes/lajidata.tsx

```diff
@@ -0,0 +1,40 @@
+import { json, LoaderFunction } from "@remix-run/node";
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const url = new URL(request.url);
+  const bounds = url.searchParams.get("bounds");
+  const accessToken = process.env.ACCESS_TOKEN;
+
+  if (!bounds || !accessToken) {
+    console.error("Missing bounds or access token.");
+    return json(
+      { data: [], message: "Missing required bounds or access token" },
+      { status: 400 },
+    );
+  }
+
+  const [southLat, northLat, westLng, eastLng] = bounds.split(":");
+
+  try {
+    const response = await fetch(
```

Comment:
Seems like there's quite a big risk of doing some kind of injection attack with this. Make sure to sanitize the input.

Reply:
Added sanitization.